advanced_filters/admin.py

```diff
@@ -77,9 +112,36 @@ def save_advanced_filter(self, request, form):
         elif request.method == "POST":
             logger.info('Failed saving advanced filter, params: %s', form.data)
 
+    def get_advanced_filter_data(self, request):
+        if request.POST.get('action') == 'advanced_filters':
+            return request.POST, True
+
+        search_query = request.GET.get('_aquery')
+        if search_query:
+            query_serializer = QSerializer(base64=True)
+            raw_query = query_serializer.loads(search_query, raw=True)
+            query_list = query_serializer.get_field_values_list(raw_query)
+
+            data = {
+                'form-TOTAL_FORMS': 0,
+                'form-INITIAL_FORMS': 0,
+                'title': 'unsaved filter'
+            }
+
+            for idx, query in enumerate(query_list):
+                query_dict = AdvancedFilterQueryForm._parse_query_dict(query, self.model)
+
+                for key, value in query_dict.items():
+                    data['form-{idx}-{key}'.format(idx=idx, key=key)] = value
+
+                data['form-TOTAL_FORMS'] += 1
+
+            return data, False
```

Comment:
The above means to initialize the forms when querying without saving? If so, perhaps we can use `AdvancedFilterForm.initialize_form` to make this more cleaner? Or is it impossible?